app/pages/stacking/delegated-stacking/components/delegated-stacking-info-card.tsx

```diff
@@ -30,10 +33,21 @@ interface PoolingInfoCardProps extends FlexProps {
 
 export const PoolingInfoCard: FC<PoolingInfoCardProps> = props => {
   const { amount, delegationType, poolStxAddress, durationInCycles, burnHeight, ...rest } = props;
-  const amountToBeStacked = stxToMicroStx(parseNumericalFormInput(amount)).integerValue();
 
   const calcFee = useCalculateFee();
 
+  const amountToBeStacked = useMemo(
+    () => stxToMicroStx(parseNumericalFormInput(amount)).integerValue(),
+    [amount]
+  );
+
+  const humanReadableAmount = useMemo(() => {
+    if (amountToBeStacked.isGreaterThan(UI_IMPOSED_MAX_STACKING_AMOUNT_USTX)) {
+      return '—';
```

Comment:
Is just displaying this enough feedback in the UI? Is that pretty standard and I just don't know it?

Reply:
A more descriptive error in the card would be better, but given the form validation and fact this is an edge case of spamming number in a field I think it works. Em dash really just indicates null value